external-crates/move/crates/move-compiler/tests/move_2024/typing/macro_duck_typing_method.exp

```diff
@@ -0,0 +1,9 @@
+error[E03006]: unexpected name in this position
+   ┌─ tests/move_2024/typing/macro_duck_typing_method.move:18:9
+   │
+18 │         $x.foo()
+   │         ^^ Macro parameters are not allowed to appear in paths
```

Comment:
Oh wait, this ain't good. Or is it? Thoughts here?

Maybe we should suggest they first let-bind the argument, e.g. "Try binding the argument to a local variable first, e.g. 'let x = $x;'

Reply:
@cgswords thought on adding this to the note?